build.sbt

```diff
@@ -150,7 +150,8 @@ lazy val example = (project in file("example"))
       akkaTestkit.map(_ % "test") ++
       akkaPersistance ++
       logback ++
-      exampleDependencies
+      exampleDependencies ++
```

Comment:
Not a critical one: 

I think that we should either get rid of "exampleDependencies" and just paste the dependencies here or add "zio" to example dependencies. I think the former feels nicer to me

Reply:
@lgajowy, refactored in this commit:
https://github.com/ScalaConsultants/mesmer/pull/470/commits

I ended up going with the `get rid of "exampleDependencies" and just paste the dependencies here` suggestion 🙂